node/cn/backend.go

```diff
@@ -549,6 +556,7 @@ func (s *CN) SetupKaiaxModules(ctx *node.ServiceContext, mValset valset.ValsetMo
 			NodeKey:       ctx.NodeKey(),
 			Chain:         s.blockchain,
 			TxPool:        s.txPool,
+			MinBalance:    mGaslessMinBal,
```

Comment:
How about changing this to node type and checking it inside?

Reply:
fixed 7c6b5687c